game/factions/faction.py

```diff
@@ -346,6 +346,14 @@ def apply_mod_settings(self, mod_settings: ModSettings) -> None:
             self.remove_preset("SA-23/S-300VM")
             self.remove_preset("SA-17")
             self.remove_preset("KS-19")
+        # llh mod pack
+        if not mod_settings.llhmodpack:
+            self.remove_preset("HPG_Ural_Isis_Troops")
```

Comment:
`remove_preset` is the wrong function for it. What you're looking for is `remove_vehicle`. 
The remove preset will remove all vehicles coming into the faction by the use of "preset_group". This is for example used for complex sam systems which include a set of vehicles.

Reply:
resolved